java/src/jmri/jmrit/timetable/configurexml/TimeTableXml.java

```diff
@@ -503,6 +503,10 @@ public static String getFileLocation() {
             }
             return fileLocation;
         }
+
+        public static void resetFileLocation() {
```

Comment:
It would be good with a comment that this method is used by tests.

Reply:
Thanks, I initially had the comment but it got lost!